app/assets/javascripts/discourse/tests/acceptance/composer-image-preview-test.js

```diff
@@ -70,6 +70,8 @@ acceptance("Composer - Image Preview", function (needs) {
       "[img]/images/avatar.png[/img]",
       // 12 Image with data attributes
       "![test|foo=bar|690x313,50%|bar=baz](upload://test.png)",
+      // 13 Video with dimensions - should not work
+      "![SampleVideo_1280x720|video](upload://test.mp4)",
```

Comment:
I wouldn't mix a negative test with positive ones, especially since we rely on a count of `.button-wrapper` elements (it's theoretically possible that video shows the element and some other regressed 😉)

You could add something like this at the end of this test instead:

```js
await fillIn(".d-editor-input", "![SampleVideo_1280x720|video](upload://test.mp4)");
assert.dom(".button-wrapper").doesNotExist();
```